src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const totalRentalCost = days * RENT_COST;
+
+  if (days >= LONG_TERM) {
+    return totalRentalCost - LONG_TERM_DISCOUNT;
```

Comment:
The use of `if` with `return` is good because it simplifies the flow of the function by exiting early when the condition is met.